extide/gradle/src/org/netbeans/modules/gradle/execute/AbstractGradleExecutor.java

```diff
@@ -35,7 +35,6 @@
 import org.openide.util.NbBundle;
 import org.openide.util.RequestProcessor;
 
-import static org.netbeans.modules.gradle.execute.Bundle.*;
```

Comment:
Javadoc on JDK-11 seems to have problems with `Bundle.*` imports - then it generates no HTML pages. I had to eliminate the import and use `Bundle.xyz` directly. Btw. the same problem appears in `maven` project, but there is too many mistakes for me to fix - could we have a transformation to avoid using `import static`, please @jlahoda, @dbalek?

Reply:
jackpot rule language can't trigger on imports as far as I know. This would have to be written by hand I believe.